CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ $ cd arturo-stac-api
 $ pip install -e .[dev]
 ```
 
-**Python3.7 only**
+**Python3.8 only**
```

Comment:
This means `pre-commit` won't work anymore if your global Python is 3.7?

Reply:
the language in pre-commit is set to python 3.8 so I guess it's just better to use python 3.8 🤷‍♂️